README.md

```diff
@@ -7,3 +7,93 @@ Tool for converting specialized annotated Verilog models into XML needed for
 
 Documentation can be found at https://python-symbiflow-v2x.readthedocs.io/en/latest/
 
+## Installation
+v2x can be installed from a local git repository using pip.
+
+```
+cd python-symbiflow-v2x
+pip install .
+```
+
+Alternatively, it can be installed from GitHub directly.
+
+```
+pip install git+https://github.com/SymbiFlow/python-symbiflow-v2x.git
```

Comment:
Does this need a `#egg=python-symbiflow-v2x` on the end?

Reply:
Currently `pip install` works without it, but seems to be the common way of doing. I'll add them later.